networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc

```diff
@@ -6,26 +6,33 @@ include::_attributes/common-attributes.adoc[]
 // Unique attribute for the version of the release notes.
 
 toc::[]
+ 
+As a cluster administrator, you can migrate to the OVN-Kubernetes network plugin from the OpenShift software-defined networking (SDN) plugin. 
 
-As a cluster administrator, you can migrate to the OVN-Kubernetes network plugin from the OpenShift SDN network plugin. The following options are available for migration:
+[WARNING]
+====
+If your cluster uses the OpenShift SDN network plugin, you cannot upgrade your cluster to {product-title} 4.17. You must initially migrate your cluster to the OVN-Kubernetes plugin and then upgrade your cluster to {product-title} 4.17.
+
+Additionally, if your cluster uses a third-party container network interface (CNI) plugin, you cannot migrate your cluster to the OVN-Kubernetes network plugin. For a complete list of these CNI plugins, see link:https://access.redhat.com/articles/5436171[Certified OpenShift CNI Plug-ins] (Red{nbsp}Hat Knowledgebase).
+====
+
+The following methods exist for migrating from the OpenShift SDN network plugin to the OVN-Kubernetes plugin:
 
 Limited live migration (preferred method)::
-This is an automated process that migrates your cluster from OpenShift SDN to OVN-Kubernetes without service interruption.
+This is an automated process that migrates your cluster from OpenShift SDN to OVN-Kubernetes without service interruption. You cannot use this method for an OpenShift SDN plugin that uses one of the Egress IP modes.
```

Comment:
Please delete "You cannot use this method for an OpenShift SDN plugin that uses one of the Egress IP modes."  Other service limitations are described in https://docs.google.com/document/d/1Nab0IuP1YrWbI5hZk8IGTDT2z-0Le2hCp64Q71fCV-I/edit?tab=t.0#heading=h.q4huypfy3ofa. I just want to say the statement of "without service interruption" is not the accurate statement